code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -183,7 +183,10 @@
 		M.adjustFireLoss(-2*REM, FALSE)
 	else
 		M.adjustFireLoss(-1.25*REM, FALSE)
-	M.adjust_bodytemperature(rand(-25,-5)*(TEMPERATURE_DAMAGE_COEFFICIENT*REM), 50)
+	M.adjust_bodytemperature(rand(-25,-5) * (TEMPERATURE_DAMAGE_COEFFICIENT*REM), 50)
+	if(ishuman(M))
+		var/mob/living/carbon/human/humi = M
+		humi.adjust_coretemperature(rand(-25,-5) * (TEMPERATURE_DAMAGE_COEFFICIENT*REM), 50)
```

Comment:
I'm seeing a lot of this sort of pattern, did you do it for every adjust_bodytemperature?

Reply:
This is for items that are intended to target the body temp for a medical reason.
Items like chilis only intend to effect skin temp, but as core temp is human only and body temp is meant to continue acting like the old system.
Only some minor changes were needed for humans in situations like this.